packages/components/release.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+if [ "$1" != "major" ] && [ "$1" != "minor" ] && [ "$1" != "patch" ];
+then
+    echo "Could not release!"
+    echo "Usage: 'npm run release -- (major|minor|patch)'"
+    echo ""
+    exit 1
+fi
+
+npm version $1
+git add .
+export NPM_VERSION=$(npm pkg get version --workspaces=false | tr -d \")
+git commit -m 'Release Components'
```

Comment:
Did you try something like this to get the version name into the commit message?
https://stackoverflow.com/a/61708683

Reply:
@ammapspeople Great find; added it now: https://github.com/MapsPeople/web-ui/pull/186/commits/3d42b79afa0942dccbc7e7ffb95f324cbc42e2e4